### PR TITLE
End2end wip

### PR DESCRIPTION
**Pull request by nforbus**
Modified the poc/Makefile to include tests

### CODE REVIEW

File: tests/diff_testing.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import sys
+# Code from: https://www.opentechguides.com/how-to/article/python/58/python-file-comparison.html
+# Ask the user to enter the names of files to compare
+
+#fname1 = input("Enter the first filename: ")
```
Comment (connorkuehl) [inline tests/diff_testing.py]: Safe to delete this dead code here.

**Comment by tim-pugh** [comment]
Before I review this I would like more information on what this intends to do. We can discuss this during the next meeting.

**Comment by tim-pugh** [comment]
I suspect that this is in regards to issue #84 , in which case perhaps we can see a demo before we merge this to see if it's useful being merged.

**Comment by tim-pugh** [comment]
Looks good! Demoed!

**Comment by nforbus** [comment]
Made the makefile test commands provide variable names.

**Comment by tim-pugh** [comment]
I'm going to close this PR since we've migrated over to the new repo. Sorry gang : - )

**Pull request closed without merging**